lib/hanami/generators/app/application.rb.tt

```diff
@@ -85,20 +94,24 @@ module <%= config[:classified_app_name] %>
       #
       # default_request_format :html
 
-      # Default format for responses that doesn't take into account the request format
+      # Default format for responses that doesn't take into account the request
+      # format
+      #
       # Argument: A symbol representation of a mime type, default to :html
       #
       # default_response_format :html
 
       # HTTP Body parsers
       # Parse non GET responses body for a specific mime type
-      # Argument: Symbol, which represent the format of the mime type (only `:json` is supported)
+      # Argument: Symbol, which represent the format of the mime type
+      #             (only `:json` is supported)
       #           Object, the parser
```

Comment:
Can we clarify what this means? I interpret it means that you can pass an Object which parses the body, but it's not super clear